ocaml/typing/typedtree.mli

```diff
@@ -869,6 +869,7 @@ and core_type_desc =
   | Ttyp_arrow of arg_label * core_type * core_type
   | Ttyp_tuple of (string option * core_type) list
   | Ttyp_constr of Path.t * Longident.t loc * core_type list
+  | Ttyp_app of core_type * core_type list
```

Comment:
FWIW I'd normally leave this part out until the PR where you fill in the code for `typetexp.ml`. It does no harm to leave it in now though.